DCode.Models/ResponseModels/Task/Task.cs

```diff
@@ -22,6 +22,7 @@ public class Task : TaskSummary, IViewModel
         public string Duration { get; set; }
         public bool IsApplied { get; set; }
         public bool IsRecommended { get; set; }
+        public int? SelectedProfType { get;set; }

```

Comment:
Please rename this as `SelectedProficiencyType` and make associated refactorings across.